up42/storage.py

```diff
@@ -236,15 +255,20 @@ def get_orders(
         if sortby not in allowed_sorting_criteria:
             raise ValueError(f"sortby parameter must be one of {allowed_sorting_criteria}!")
         sort = f"{sortby},{'desc' if descending else 'asc'}"
-        url = f"{self.auth._endpoint()}/workspaces/{self.workspace_id}/orders?format=paginated&sort={sort}"
-        if order_type is not None:
-            if order_type in ["TASKING", "ARCHIVE"]:
-                url += f"&type={order_type}"
-            else:
-                logger.warning("order_type should be TASKING or ARCHIVE. Ignoring this filter.")
-        if tags is not None:
-            for tag in tags:
-                url += f"&tags={tag}"
+        base_url = f"{self.auth._endpoint()}/v2/orders"
+
+        params = {
+            "sort": sort,
+            "workspaceId": self.workspace_id if workspace_orders else None,
+            "displayName": name,
+            "type": order_type if order_type in ["TASKING", "ARCHIVE"] else None,
+            "tags": tags,
+            "status": [s for s in status if s in allowed_status] if status else None,
+        }
+        params = {k: v for k, v in params.items() if v is not None}
+
+        url = urljoin(base_url, "?" + urlencode(params, doseq=True, safe=""))
```

Comment:
Let's add a Jira task on auth._request to allow to pass parameters to avoid this in the future and rely on requests param handling